packages/components/src/Components/ConditionalFilter/Group.js

```diff
@@ -193,24 +229,17 @@ Group.propTypes = {
         value: PropTypes.string,
         onSelect: PropTypes.func,
         type: PropTypes.oneOf(Object.values(groupType)),
-        items: PropTypes.arrayOf(
-            PropTypes.shape({
-                value: PropTypes.string,
-                label: PropTypes.node,
-                id: PropTypes.string,
-                isChecked: PropTypes.bool,
-                onClick: PropTypes.func,
-                props: PropTypes.shape({
-                    [PropTypes.string]: PropTypes.any
-                })
-            })
-        )
-    }))
+        items: itemsProps
+    })),
+    items: itemsProps,
+    isFilterable: PropTypes.bool
```

Comment:
Can you also add `onFilter` prop as well? No need to add default props as we will expect it will be undefined by default.
```JSX
onFilter: PropTypes.func
```